<launch>
    <arg name="with_rviz" default="true"/>

    <group ns="dmu_node">
        <param name="device" value="/dev/ttyUSB0"/>
        <param name="frame_id" value="imu"/>
        <param name="rate" value="200"/>
    </group>

    <node name="dmu_node" pkg="dmu_ros" type="dmu_node"/>

</launch>
